src/elements/TableProducerCell/TableProducerCell.js

```diff
@@ -75,6 +75,7 @@ TableProducerCell.defaultProps = {
 TableProducerCell.propTypes = {
   bodyCellClassName: PropTypes.string,
   className: PropTypes.string,
+  id: PropTypes.string,
```

Comment:
the prop must be required or has the default value

Reply:
@mavdryk added added required  attribute to 'Id' prop.